apps/judicial-system/web/src/routes/Court/Indictments/Conclusion/Conclusion.strings.ts

```diff
@@ -0,0 +1,84 @@
+import { defineMessages } from 'react-intl'
+
+export const strings = defineMessages({
+  title: {
+    id: 'judicial.system.core:court_record.title_v1',
```

Comment:
Please change the id's from court_record to conclusion

Reply:
I think that could potentially cause a few issues. If the strings have been changed in Contentful and we change the id, wouldn't we then display the default message instead of the string in Contentful, since we'd essentially be creating a new string?

Reply:
I agree with @unakb. I will make sure to check to strings in Contentful and make necessary changes if necessary.